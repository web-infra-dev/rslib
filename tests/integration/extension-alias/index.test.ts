import { buildAndGetResults } from 'test-helper';
import { expect, test } from 'vitest';

test('resolve.extensionAlias should work', async () => {
  const fixturePath = __dirname;
  const { entries } = await buildAndGetResults(fixturePath);

  expect(entries.cjs).toMatchSnapshot();
  expect(entries.esm).toMatchSnapshot();
});
