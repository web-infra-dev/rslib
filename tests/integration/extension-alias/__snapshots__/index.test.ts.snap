// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`resolve.extensionAlias should work 1`] = `
""use strict";
var __webpack_exports__ = {};
const bar = 'bar';
const foo = 'foo';
console.log(foo + bar);
for(var __webpack_i__ in __webpack_exports__)exports[__webpack_i__] = __webpack_exports__[__webpack_i__];
Object.defineProperty(exports, '__esModule', {
    value: true
});
"
`;

exports[`resolve.extensionAlias should work 2`] = `
"const bar = 'bar';
const foo = 'foo';
console.log(foo + bar);
"
`;
